sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/SearchIndexingBufferedSenderOptions.java

```diff
@@ -123,38 +127,96 @@ public Duration getAutoFlushWindow() {
      *
      * @return The number of documents required before a flush is triggered.
      */
-    int getBatchSize() {
-        return (batchSize == null) ? DEFAULT_BATCH_SIZE : batchSize;
+    public int getInitialBatchActionCount() {
```

Comment:
Do we need to hide it here as well?

Reply:
This is the options class passed into the buffered sender, it could be made package private but I was copying @tg-msft design where all properties had public get and set.